Automattic/studio
# Studio: Add prompt limit UI

Fixes https://github.com/Automattic/dotcom-forge/issues/7409

## Proposed Changes

This diff, adds a prompt limit UI into user settings

## Testing Instructions
1. Start studio `nvim use && STUDIO_AI=true npm start`
3. Ensure you are logged in
4. Navigate to settings and ensure that you see the following UI screen
![2024-05-31T10:25:46,590050296+03:00](https://github.com/Automattic/studio/assets/497103/f29481ef-96f0-4b5d-8555-a50a71a7be7d)
5. Start studio without STUDIO_AI feature enabled. Ensure that the new UI is not shown.





## Pre-merge Checklist
